source/search_page/assets/script.js

```diff
@@ -23,4 +21,32 @@ function createRecipeCards() {
     
   
 }
+*/
+
+async function searchFetchRecipes(searchBar) {
+  let searchString = `https://api.spoonacular.com/recipes/complexSearch?apiKey=03722052291e4f84bce1021acd82624f&${searchBar}&addRecipeInformation=True`;
+  console.log(searchString)  
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8435053)
```suggestion
  console.log(searchString);  
```